### ISSUE TITLE
Open PRs with Packit config for new Fedora packages 

### ISSUE THREAD

**Issue by lbarcziova** [packit/packit-service#2506]
This is a follow-up of the discussion on Flock, where this idea came up.

TODO:
- [ ] listen to the messages about new package repository in dist-git (check which one is the best fit, e.g. [pagure.project.new](https://fedora-fedmsg.readthedocs.io/en/latest/topics.html#pagure-project-new))
   - if we can’t be sure the repo is prepared for accepting the PRs in that moment, implement scheduling the task later (e.g. after 24h)

- [ ] open PRs for those
   - description should contain brief introduction to Packit and downstream automation capabilities
   - PR should contain a default Packit configuration (created by `packit dist-git init`)

**Comment by lachmanfrantisek** [comment]
Here's a list of issues tracking new repositories being created so we can see the frequency:
https://pagure.io/releng/fedora-scm-requests/issues?status=all&search_pattern=new+repo&close_status=

**Comment by nforro** [comment]
We decided to make this a Fedora Change, [here](https://github.com/packit/packit-service/issues/2548) is an issue to file a Change Proposal.

**Comment by lbarcziova** [comment]
One note to the functionality itself we probably haven't considered so far, we should probably also handle adding the project to release-monitoring.org as this is now not automatic for new packages, and also enabling the monitoring.

**Comment by lbarcziova** [comment]
In case we will still be running in the current cluster when implementing this, maybe we should also automatically get and check the sources of the newly created package, whether it is not blocked by the firewall (and enable it in case yes?).

**Comment by nforro** [comment]
fedpkg PR: https://pagure.io/fedpkg/pull-request/586

**Comment by nforro** [comment]
TODO:

- [ ] Implement a new task in Packit Service triggered by a webhook
- [ ] Set up a shared webhook secret in Packit Service and in [Fedora Infra](https://pagure.io/fedora-infra/ansible/blob/main/f/roles/openshift-apps/poddlers) for the `scm-request-processor` poddler
- [ ] Implement sending the webhook from [`SCMRequestProcessor.process_new_repo()`](https://pagure.io/fedora-infra/toddlers/blob/main/f/toddlers/plugins/scm_request_processor.py#_432)
